internal/controller/assets/postgres_bootstrap.sh

```
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+cat /var/lib/pgsql/data/userdata/postgresql.conf
+
+echo "attempting to create llama-stack database and pg_trgm extension if they do not exist"
+
+_psql () { psql --set ON_ERROR_STOP=1 "$@" ; }
+
+# Create database for llama-stack conversation storage
+DB_NAME="llamastack"
+
+echo "SELECT 'CREATE DATABASE $DB_NAME' WHERE NOT EXISTS (SELECT FROM pg_database WHERE datname = '$DB_NAME')\gexec" | _psql -d $POSTGRESQL_DATABASE
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)